### PR TITLE
Add About Page

### PR DESCRIPTION
**Pull request by piyush-pawar-17**

### CODE REVIEW

File: src/components/Social.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import Facebook from '../img/facebook.svg';
+import Instagram from '../img/instagram.svg';
+import Twitter from '../img/twitter.svg';
+
+const Social = () => (
+	<div className="social-icons">
+		<Link to="/">
```
Comment (Monal5031) [inline src/components/Social.js]: links:
fb: https://www.facebook.com/cerebro.iiitv
insta: https://www.instagram.com/cerebro.iiitv/
twitter: https://twitter.com/Cerebro_iiitv

**Review by aashutoshrathi — COMMENTED** [review]
Deploy link. Please :pray:

File: src/components/Social.js (hunk 1 above)
Comment (piyush-pawar-17) [reply src/components/Social.js]: Added

**Review by anshumanv — APPROVED** [review]
Checked locally, not sure why netlify builds aren't running 🤔

**Pull request merged**